modules/oc-mirror-configuring-credentials-v2.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-configuring-credentials-v2_{context}"]
+= Configuring credentials that enable image mirroring
```

Comment:
This file looks almost identical to the existing module https://github.com/openshift/openshift-docs/blob/main/modules/installation-adding-registry-pull-secret.adoc

Instead of copying and pasting this as a new module, you should use that file and then just add conditionals if there are any differences you need to make for this v2 oc-mirror